src/TagBot/TagBot.jl

```diff
@@ -45,6 +45,7 @@ function main()
 end
 
 function repo_and_version_of_pull_request_body(body)
+    isnothing(body) && return (nothing, nothing)
```

Comment:
Can you add a brief comment explaining when we hit this code path (i.e. we hit this code path if the PR body is empty).

Reply:
I had been going back and forth on that and ended up removing it in the end 😅 Added back in 3fbe6e210c591c01596de92d6911f7a86c9eda97.